### PR TITLE
Release v0.37.0

### PR DESCRIPTION
**Pull request by kddejong**
*Issue #, if available:*

*Description of changes:*
- Release v0.37.0

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+### v0.36.2
+###### CloudFormation Specifications
+- Update CloudFormation specs to 18.5.0 (pull #[1715](https://github.com/aws-cloudformation/cfn-python-lint/pull/1715))
+- Get `AllowedValues` from Botocore (pull #[1682](https://github.com/aws-cloudformation/cfn-python-lint/pull/1682))
```
Comment (PatMyron) [inline CHANGELOG.md:4]:
```suggestion
- Get `AllowedValues` from Botocore during `--update-specs` (pull #[1682](https://github.com/aws-cloudformation/cfn-python-lint/pull/1682))
```
think this change is fundamental enough to at least bump minor version to `0.37.0` as well

Comment (PatMyron) [reply CHANGELOG.md:4]: Should https://github.com/aws-cloudformation/cfn-python-lint/pull/1716 be included in the changelog as well?

**Comment by PatMyron** [comment]
Should [`AWS::StepFunctions::Activity`](https://github.com/aws-cloudformation/cfn-python-lint/pull/1715#discussion_r496397195) be fixed before releasing?

File: CHANGELOG.md (hunk 1 above)
Comment (kddejong) [reply CHANGELOG.md:4]: Yes.  I'll add it

**Review by PatMyron — APPROVED** [review]

**Pull request merged**